dali/daliadmin/daadmin.hpp

```diff
@@ -23,13 +23,17 @@
 
 #include "dautils.hpp"
 
+namespace daadmin
+{
+
 extern DALIADMIN_API void setDaliConnectTimeoutMs(unsigned timeoutMs);
 extern DALIADMIN_API void xmlSize(const char *filename, double pc);
 extern DALIADMIN_API void translateToXpath(const char *logicalfile, DfsXmlBranchKind tailType = DXB_File);
 
 extern DALIADMIN_API void _export_(const char *path, const char *dst, bool safe = false);
```

Comment:
should rename this and any named like _xx_

Reply:
Will try

Reply:
@jakesmith looks like the 'export' is causing similar problems for the compiler. Rename it to 'save'?

Reply:
exportToFile would be appropriate here ('dst' should be renamed 'filename' for clarity).
Arguably there should also be an exportToXML as well, so symmetrical with importFromXML/importFromFile